src/index.tsx

```diff
@@ -220,7 +229,13 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
             nonce,
         } = this.props;
 
-        const contentEl = content();
+        let contentEl = lazyOption && typeof lazyOption.content === "function" ? lazyOption.content() : null;
```

Comment:
```suggestion
        let contentEl = typeof lazyOption?.content === "function" ? lazyOption.content() : null;
```

Reply:
This is a very good suggestion. I will modify accordingly